frontend/Dockerfile

```diff
@@ -13,7 +13,11 @@ COPY . .
 
 RUN cd  /frontend
 RUN npm update & npm cache clean --force
+## FORCE SOME THINGS
```

Comment:
Please write a significant message, possibly mentionning the cve or at least the ticket linked to it.